esmvaltool/diag_scripts/hydrology/marrmot.py

```diff
@@ -79,6 +79,20 @@ def _get_extra_info(cube):
     return time_start_end, lat_lon
 
 
+def fix_era5_time_coordinate(cube):
```

Comment:
```suggestion
def shift_era5_time_coordinate(cube):
```
Because actually it's not a fix, but rather a workaround to make the variables with different time points compatible.